### DISCUSSION TITLE
Colormap removal API

### DISCUSSION THREAD

**Discussion by ToppDev** [epezent/implot#341] [Ideas]
Hello,

I would like to request a feature for colormap handling.

### Problem to solve 

Currently it is possible to add colormaps via `AddColormap(...)`, but it is not possible to remove them.

### Proposal

Add two new functions into the API

```C++
void RemoveColormap(const char* name);

void RemoveColormap(ImPlotColormap colormap);
```

### Further details

The style editor could also be expanded to showcase this functionality.

**Comment by epezent** [comment]
I think this is fine to add. We have to decide weather or not to allow removing built in color maps. My inclination is that it should be an error and cause an assert. 

Would you consider submitting a PR for this feature?

**Comment by ToppDev** [comment]
Throwing an error for the built in color maps sounds fine to me.

I will give it a try this weekend and make a pull request if ready.

**Comment by ToppDev** [comment]
I implemented it, see #344 . I am actually unfamiliar with Discussions on GitHub. Should I consider making this into an issue now?
